### PR TITLE
DTD-200 : Added extra edge cases and update branch with rates change for 2 payment windows

### PR DESCRIPTION
**Pull request by thadeusssewanyana**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/api/cucumber/runner/InterestForecastingApiTestRunner.scala [hunk 1]
```diff
@@ -24,6 +24,6 @@ import org.junit.runner.RunWith
   features = Array("src/test/resources/features/ifs"),
   glue = Array("uk.gov.hmrc.test.api.cucumber.stepdefs.ifs", "uk.gov.hmrc.test.api.cucumber.hooks"),
   plugin = Array("pretty", "html:target/cucumber", "json:target/cucumber.json"),
-  tags = Array("")
+  tags = Array("@smoke")
```
Comment (rogerball) [inline src/test/scala/uk/gov/hmrc/test/api/cucumber/runner/InterestForecastingApiTestRunner.scala]: remove

**Review by rogerball — APPROVED** [review]

**Review by kiranpl — APPROVED** [review]
